nixos/modules/services/web-apps/lasuite-docs.nix

```
@@ -0,0 +1,473 @@
+        Address used by gunicorn to listen to.
+      '';
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 8000;
+      description = ''
+        Port used by gunicorn to listen to.
+      '';
+    };
+
+    enableNginx = mkOption {
+      type = types.bool;
+      default = true;
+      description = ''
+        Enable Nginx as a proxy server.
+      '';
+    };
+
+    secretKeyPath = mkOption {
+      type = types.nullOr types.path;
+      default = null;
+      description = ''
+        Path to the Django secret key.
+      '';
+    };
+
+    s3 = {
+      url = mkOption {
+        type = types.str;
+        description = ''
+          URL of the s3 bucket.
+        '';
+      };
+
+      accessKeyIDPath = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        description = ''
+          Path to the access key ID of the bucket.
+        '';
+      };
+
+      secretAccessKeyPath = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        description = ''
+          Path to the secret access key of the bucket.
+        '';
+      };
+    };
+
+    oidc = {
+      clientSecretPath = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        description = ''
+          Path to the client secret of the client of OIDC.
+        '';
+      };
+    };
+
+    database = {
+      createLocally = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Configure local PostgreSQL database server for docs.
+        '';
+      };
+    };
+
+    redis = {
+      createLocally = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Configure local Redis cache server for docs.
+        '';
+      };
+    };
+
+    collaborationServer = {
+      package = mkPackageOption pkgs "lasuite-docs-collaboration-server" { };
+
+      port = mkOption {
+        type = types.port;
+        default = 4444;
+        description = ''
+          Port used by the collaboration server to listen.
+        '';
+      };
+
+      serverSecretPath = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        description = ''
+          Path to the server secret of the collaboration server.
+        '';
+      };
+
+      yproviderApiKeyPath = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        description = ''
+          Path to the y-provider API key.
+        '';
+      };
+
+      config = mkOption {
+        type = types.attrsOf (
+          types.oneOf [
+            types.str
+            types.bool
+          ]
+        );
+        default = { };
+        example = ''
+          {
+            COLLABORATION_LOGGING = true;
+          }
+        '';
+        description = ''
+          Configuration options of collaboration server.
+
+          See https://github.com/suitenumerique/docs/blob/v3.1.0/docs/env.md
```

```suggestion
          See https://github.com/suitenumerique/docs/blob/v${cfg.collaborationServer.package.version}/docs/env.md
```

Reference the package version here, so the user can find the correct list of environment variables matching their version.